doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -633,6 +633,8 @@ Scripts
 
 This section provides detailed lists of changes by :ref:`script <scripts>`.
 
+* Added new ``west sbom`` command that generates :ref:`Software bill of materials <sbom_script>`.
```

Comment:
Various places in this PR (including documentation, e.g. here, but also the commit log for this patch) are still referring to `west sbom` instead of `west ncs-sbom`.

The explanation of why this is needed when `west spdx` is already present is also not present in the commit log (or the documentation, as far as I can see) as discussed.

Reply:
The new commit message will be:

```
scripts: sbom: Add "west sbom" tool.

The "west ncs-sbom" tool creates a Software Bill of Materials.
It uses a built application or the specific files as an input,
detects their licenses, and creates an output report that can
be in HTML or SPDX format.

It is significantly different than the Zephyr's "west spdx"
tool, so it was implemented from the. It has a different
method of getting files from the application that gets a more
precise list of files. It has also additional methods for
verifying that the list is correct. It adds more license
detection methods including an external detector from
"scancode-toolkit". The output (both HTML and SPDX) is more
readable.
```